### ISSUE TITLE
VAMC A11y Audit (2 of 4)

### ISSUE THREAD

**Issue by r-bartlett-gsa** [department-of-veterans-affairs/va.gov-cms#23958]
## Description
Complete a full accessibility audit of the VAMC product, including:
- 1 of 4, other ticket, #17372
  - [System page](https://www.va.gov/boston-health-care/)
  - [Health Services page](https://www.va.gov/boston-health-care/health-services/)
- **2 of 4, this ticket, 23958** 
  - [Operating Status](https://www.va.gov/boston-health-care/operating-status/)
  - [Locations page](https://www.va.gov/boston-health-care/locations/)
  - [Facility page](https://www.va.gov/boston-health-care/locations/jamaica-plain-va-medical-center/)
  - [Policies](https://www.va.gov/boston-health-care/policies/)
  - [Register for Care](https://www.va.gov/boston-health-care/register-for-care/)
  - [Billing and Insurance](https://www.va.gov/boston-health-care/billing-and-insurance/)
  - [Medical Records](https://www.va.gov/boston-health-care/medical-records-office/)
  - [Leadership](https://www.va.gov/boston-health-care/about-us/leadership/)
  - [Staff Details](https://www.va.gov/boston-health-care/staff-profiles/vincent-ng/)
- 3 of 4, other ticket, 
  - [News Releases](https://www.va.gov/boston-health-care/news-releases/)
  - [News Releases Details](https://www.va.gov/boston-health-care/news-releases/va-boston-housed-more-than-350-homeless-veterans-in-2023/)
  - [Stories](https://www.va.gov/boston-health-care/stories/)
  - [Story Details](https://www.va.gov/boston-health-care/stories/comcasts-veteran-network-and-va-boston-unveil-haven-for-veteran-families/)
- 4 of 4, other ticket, 
  - VAMC Detail pages
    - [Make an Appointment](https://www.va.gov/boston-health-care/make-an-appointment/)
    - [Contact Us](https://www.va.gov/boston-health-care/contact-us/)
    - [About Us](https://www.va.gov/boston-health-care/about-us/)
    - [Work with Us](https://www.va.gov/boston-health-care/work-with-us/)

## Acceptance Criteria

- [ ] Accessibility audit has been completed
- [ ] Tickets have been created for recommended updates and improvements
- [ ] Any high priority/critical updates are brought to the attention of PM/DM/PO

**Comment by qjac** [comment]
### [Operating Status page](https://www.va.gov/boston-health-care/operating-status/)

- [x] Run axeDev tools on the page and confirm that either:
  - [x] The page results in zero flagged issues OR
  - [ ] If existing issues are found, confirm that they are already ticketed in the Sitewide backlog

<img width="419" height="382" alt="Image" src="https://github.com/user-attachments/assets/c1be2f52-ccde-4e6d-9d65-5bf153d386c9" />

- [x] Use of color
  - [x] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
  - [x] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
  - [x] Non-text elements have a 3:1 contrast ratio to their background and to neighboring elements (or better)
  - [x] Color is not the only way to distinguish links from other text (eg. links are underlined)
  - [x] Any charts, maps, infographics, and tables convey all information without only relying on color
  - [x] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")
- [x] Content zoom
  - [x] All page elements are readable and usable at 200% zoom
  - [x] All page elements are readable and usable at 300% zoom
  - [x] All page elements are readable and usable at 400% zoom
- [x] Font Size
  - [x] Within your default browser, update your settings to the largest font size available (and smallest)
  - [x] Confirm that all the text increases in size and scales correctly
- [x] Keyboard navigation
  - [x] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
  - [x] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
  - [x] All elements under focus have a visible focus indicator
    - ⚠️ https://github.com/department-of-veterans-affairs/va.gov-cms/issues/24030
  - [x] The order of [Tab] stops made sense and was appropriate for completing tasks
- [x] Code quality review
  - [x] Confirm that any Design System components are used as intended, according to Design System documentation
    - [x] The 'Patient resources' in the 'Emergency information' are not using va-link or va-telephone because that content comes from a WYSIWIG.  
  - [x] Confirm that all HTML elements are implemented consistent with specifications and best practices, including:
    - [x] Element IDs are unique
    - [x] Links are navigational and are coded using the a element
    - [x] Buttons are actions and are coded using the button element
    - [x] Form input are properly labeled and, when appropriate, grouped in a fieldset with related inputs
    - [x] Visible input labels match the name attribute
    - [x] Form inputs have the appropriate input type attribute so that the purpose of the input can be determined programmatically (when applicable)
    - [x] Status messages have the appropriate role attribute
  - [x] Confirm that all ARIA attributes are implemented according to the W3C’s ARIA Authoring Practices Guide and not used unnecessarily
- [x] Screen reader testing
  - [x] Confirm that all page content is announced by the screen reader, with the appropriate context and ordering to complete tasks.

**Comment by qjac** [comment]
[Locations page](https://www.va.gov/boston-health-care/locations/)

- [x] Run axeDev tools on the page and confirm that either:
  - [x] The page results in zero flagged issues OR
  - [x] If existing issues are found, confirm that they are already ticketed in the Sitewide backlog
  
<img width="513" height="412" alt="Image" src="https://github.com/user-attachments/assets/63259756-5c18-4573-b6d7-4e419e5caa1d" />

- [x] Use of color
  - [x] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
  - [x] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
  - [x] Non-text elements have a 3:1 contrast ratio to their background and to neighboring elements (or better)
  - [x] Color is not the only way to distinguish links from other text (eg. links are underlined)
  - [x] Any charts, maps, infographics, and tables convey all information without only relying on color
  - [x] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")
- [x] Content zoom
  - [x] All page elements are readable and usable at 200% zoom
  - [x] All page elements are readable and usable at 300% zoom
  - [x] All page elements are readable and usable at 400% zoom
- [x] Font Size
  - [x] Within your default browser, update your settings to the largest font size available (and smallest)
  - [x] Confirm that all the text increases in size and scales correctly
    - https://github.com/department-of-veterans-affairs/va.gov-cms/issues/24191
- [x] Keyboard navigation
  - [x] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
  - [x] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
  - [x] All elements under focus have a visible focus indicator
  - [x] The order of [Tab] stops made sense and was appropriate for completing tasks
- [x] Code quality review
  - [x] Confirm that any Design System components are used as intended, according to Design System documentation
  - [x] Confirm that all HTML elements are implemented consistent with specifications and best practices, including:
    - [x] Element IDs are unique
      - https://github.com/department-of-veterans-affairs/va.gov-cms/issues/24178
    - [x] Links are navigational and are coded using the a element
    - [x] Buttons are actions and are coded using the button element
    - [x] Form input are properly labeled and, when appropriate, grouped in a fieldset with related inputs
    - [x] Visible input labels match the name attribute
    - [x] Form inputs have the appropriate input type attribute so that the purpose of the input can be determined programmatically (when applicable)
    - [x] Status messages have the appropriate role attribute
  - [x] Confirm that all ARIA attributes are implemented according to the W3C’s ARIA Authoring Practices Guide and not used unnecessarily
- [x] Screen reader testing
  - [x] Confirm that all page content is announced by the screen reader, with the appropriate context and ordering to complete tasks.

**Comment by qjac** [comment]
### [Facility page](https://www.va.gov/boston-health-care/locations/jamaica-plain-va-medical-center/)

- [x] Run axeDev tools on the page and confirm that either:
  - [x] The page results in zero flagged issues OR
  - [ ] If existing issues are found, confirm that they are already ticketed in the Sitewide backlog
    <img width="601" height="475" alt="Image" src="https://github.com/user-attachments/assets/14623b58-0d21-4249-a0f4-ce325df983d2" />
- [ ] Use of color
  - [ ] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
  - [ ] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
  - [ ] Non-text elements have a 3:1 contrast ratio to their background and to neighboring elements (or better)
  - [ ] Color is not the only way to distinguish links from other text (eg. links are underlined)
  - [ ] Any charts, maps, infographics, and tables convey all information without only relying on color
  - [ ] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")
- [ ] Content zoom
  - [ ] All page elements are readable and usable at 200% zoom
  - [ ] All page elements are readable and usable at 300% zoom
  - [x] All page elements are readable and usable at 400% zoom
- [ ] Font Size
  - [ ] Within your default browser, update your settings to the largest font size available (and smallest)
  - [ ] Confirm that all the text increases in size and scales correctly
- [ ] Keyboard navigation
  - [x] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
  - [x] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
  - [ ] All elements under focus have a visible focus indicator
  - [ ] The order of [Tab] stops made sense and was appropriate for completing tasks
  - Map is linked and in mobile out of focus order. Is this the same as what was found on System page? 
- [ ] Code quality review
  - [ ] Confirm that any Design System components are used as intended, according to Design System documentation
  - [ ] Confirm that all HTML elements are implemented consistent with specifications and best practices, including:
    - [ ] Element IDs are unique
    - [ ] Links are navigational and are coded using the a element
    - [ ] Buttons are actions and are coded using the button element
    - [ ] Form input are properly labeled and, when appropriate, grouped in a fieldset with related inputs
    - [ ] Visible input labels match the name attribute
    - [ ] Form inputs have the appropriate input type attribute so that the purpose of the input can be determined programmatically (when applicable)
    - [ ] Status messages have the appropriate role attribute
  - [ ] Confirm that all ARIA attributes are implemented according to the W3C’s ARIA Authoring Practices Guide and not used unnecessarily
- [ ] Screen reader testing
  - [x] Confirm that all page content is announced by the screen reader, with the appropriate context and ordering to complete tasks.
  - [ ] Recheck telephone numbers. Many have visible labels and sr-only/aria-describedby. Also duplicate headings with duplicate phone numbers...why? What is outputting that? Also some announced with no label...are those in wysiwig and can they get an aria-describedby?
